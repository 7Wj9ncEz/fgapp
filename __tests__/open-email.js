import 'react-native';
import React from 'react';
import OpenEmail from '../components/open-email.js';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <OpenEmail/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
