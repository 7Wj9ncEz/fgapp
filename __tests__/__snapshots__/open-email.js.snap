exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Object {
          "color": "white",
          "fontSize": 16,
        },
        Object {
          "fontFamily": "FontAwesome",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
      ]
    }>
    
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    numberOfLines={1}
    style={
      Object {
        "color": "white",
        "fontSize": 16,
        "fontWeight": "bold",
        "marginLeft": 10,
      }
    }>
    Não disponível.
  </Text>
</View>
`;
